rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -3689,7 +3738,9 @@ bool ppu_interpreter::FMUL(PPUThread& ppu, ppu_opcode_t op)
 
 bool ppu_interpreter::FRSQRTE(PPUThread& ppu, ppu_opcode_t op)
 {
-	ppu.FPR[op.frd] = 1.0 / std::sqrt(ppu.FPR[op.frb]);
+	f32 value = f32(ppu.FPR[op.frb]);
+	_mm_store_ss(&value, _mm_rsqrt_ss(_mm_load_ss(&value)));
```

Comment:
This will break if games rely on weird floating-point bullshit. And they probably do.